src/app.js

```diff
@@ -7,6 +7,7 @@ import config from './load-config.js';
 
 import * as accountApp from './app-account.js';
 import * as syncApp from './app-sync.js';
+import * as nordigenApp from './../app-nordigen/app-nordigen.js';
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you move all the `app-nordigen` folder inside `src`? This would help us better organize all the code.
```suggestion
import * as nordigenApp from './app-nordigen/app-nordigen.js';
```